VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_: Precharge_Controller


BO_ 1088 PRECHARGE_CONTROLLER_HEARTBEAT: 4 Precharge_Controller
 SG_ precharge_heartbeat_state : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_heartbeat_counter : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_pcb_temperature : 16|8@1+ (1,0) [0|0] "" Vector__XXX

BO_ 1089 PRECHARGE_CONTROLLER_ERROR: 4 Precharge_Controller
 SG_ precharge_error_present : 0|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_warning_present : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_IMD_ok : 8|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_PDOC_ok : 9|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_pre_dis_mismatch : 25|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_negative_AIR_mismatch : 26|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_positive_AIR_mismatch : 27|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_PCB_overtemperature : 24|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_orion_disconnected : 10|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_cell_undervoltage : 11|1@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_cell_overvoltage : 12|1@1+ (1,0) [0|0] "" Vector__XXX

BO_ 1090 PRECHARGE_CONTROLLER_ANALOGUE: 8 Precharge_Controller
 SG_ precharge_pdoc_temperature : 0|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_pdoc_ref_temperature : 16|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_mc_voltage : 32|16@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_battery_voltage : 48|16@1+ (1,0) [0|0] "" Vector__XXX

BO_ 1091 PRECHARGE_CONTROLLER_PERIPHERAL: 6 Precharge_Controller
 SG_ precharge_IMD_period : 8|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_IMD_duty_cycle : 16|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_AIR_power : 24|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_AIR_neg_feedback : 40|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_AIR_pos_feedback : 32|8@1+ (1,0) [0|0] "" Vector__XXX
 SG_ precharge_relay : 0|8@1+ (1,0) [0|0] "" Vector__XXX




